src/utils/components/CPUDescription/utils/types.ts

```diff
@@ -0,0 +1,7 @@
+import { CpuMemHelperTextResources } from './utils';
+
+export type CpuMemHelperTextResourcesProps =
+  | typeof CpuMemHelperTextResources.ActualVM
```

Comment:
isnt all typeof CpuMemHelperTextResources.ActualVM
  | typeof CpuMemHelperTextResources.FutureVM
  | typeof CpuMemHelperTextResources.Template
  | typeof CpuMemHelperTextResources.VMI;

is just type string?

Reply:
maybe u meant to do type CpuMemHelperTextResourcesProps = keyof CpuMemHelperTextResources ?

Reply:
Yeah, that could work, too. Just haven't realized that. Let me try. And thanks!

Reply:
Soo, no, it doesn't work.

Reply:
So I think u can simply use string.. As they are string

Reply:
But if I use just `string` as a type, the scope of possible values changes dramatically. This is not what I wanted to achieve.

Reply:
what is currently will respolve to string and wont server the purpose u trying to achieve 
![image](https://github.com/kubevirt-ui/kubevirt-plugin/assets/14824964/6323cce0-c156-426c-81ab-dc0718f086c6)

even when I tried something a little bit more complicated such as 
![image](https://github.com/kubevirt-ui/kubevirt-plugin/assets/14824964/2f3bbb90-ebe0-41ff-872b-17ea74129453)

it won't work beacust of t function that resolves to string but typescript don't know to which exact string

so maybe try to search for something that will serve your purpose or just use type string.

Reply:
ok